File: frontend/src/domain/Stadion.ts
export class Stadion {
    id: number;

    naziv: string;

    kapacitet: number;

    slika: string;

    constructor(
        id: number,
        naziv: string,
        kapacitet: number,
        slika: string
    ) {
        this.id = id;
        this.naziv = naziv;
        this.kapacitet = kapacitet;
        this.slika = slika;
    }

}

File: backend/football_world_cup/src/main/java/rs/ac/bg/fon/football_world_cup/domain/Stadion.java
package rs.ac.bg.fon.football_world_cup.domain;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.Id;
import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.PositiveOrZero;
import lombok.*;

@Entity

@NoArgsConstructor
@AllArgsConstructor
@Builder
@Getter
@Setter
@EqualsAndHashCode
@ToString
public class Stadion {

    @Id
    @GeneratedValue
    private Long id;

    @NotBlank(message = "Mora postojati naziv stadiona")
    private String naziv;

    @PositiveOrZero(message = "Kapacitet mora biti pozitivan celobrojan broj")
    private int kapacitet;

    private String slika;
}
